tasks/pipeline.py

```diff
@@ -469,7 +469,7 @@ def trigger_child_pipeline(_, git_ref, project_name, variables="", follow=True):
 def parse(commit_str):
     lines = commit_str.split("\n")
     title = lines[0]
-    url = "NO_URL"
+    url = ""
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Could you explain this change ?

Reply:
this just means that if the url is not set in the if statement below (L475) it will be falsey on L525